pkg/admission/machines.go

```diff
@@ -112,3 +118,15 @@ func (ad *admissionData) defaultAndValidateMachineSpec(spec *clusterv1alpha1.Mac
 
 	return nil
 }
+
+func validatePublicKeys(keys []string) error {
+	for _, s := range keys {
+		//_, err := ssh.ParsePublicKey([]byte(s))
+		_, _, _, _, err := ssh.ParseAuthorizedKey([]byte(s))
+		if err != nil {
+			return fmt.Errorf("invalid public key '%s': %v", s, err)
```

Comment:
Nit: Use `%q` instead of `'%s' `

Reply:
Done with https://github.com/kubermatic/machine-controller/pull/497/commits/5e3e9a738530605e8e89d8c06d607719903919bb